### PR TITLE
Bump to v4.0.3

### PR DESCRIPTION
**Pull request by junderw**
After this is merged I will publish 4.0.3 to npm.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# 4.0.3
+__fixed__
+- fixed `Block.calculateTarget` to allow for exponents up to 29
+- payments p2sh now takes the network from the redeem attribute if one is not given in the object argument
+- fixed some low priority rarely occurring bugs with multisig payments and `TransactionBuilder` multisig processing
```
Comment (dcousens) [inline CHANGELOG.md]: https://github.com/bitcoinjs/bitcoinjs-lib/pull/1307

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+# 4.0.3
+__fixed__
+- fixed `Block.calculateTarget` to allow for exponents up to 29
+- payments p2sh now takes the network from the redeem attribute if one is not given in the object argument
```
Comment (dcousens) [inline CHANGELOG.md]: https://github.com/bitcoinjs/bitcoinjs-lib/pull/1232

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+- fixed some low priority rarely occurring bugs with multisig payments and `TransactionBuilder` multisig processing
+
+__added__
+- regtest network object to `networks`
```
Comment (dcousens) [inline CHANGELOG.md]: https://github.com/bitcoinjs/bitcoinjs-lib/pull/1261

**Comment by dcousens** [comment]
Missing https://github.com/bitcoinjs/bitcoinjs-lib/pull/1151

**Comment by junderw** [comment]
@dcousens Fixed.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,13 @@
+# 4.0.3
+__fixed__
+- Fixed `TransactionBuilder` to require that the Transaction has outputs before signing (#1151)
+- Payments p2sh now takes the network from the redeem attribute if one is not given in the object argument (#1232)
```
[the remark this answers is not in the document]
Comment (junderw) [reply CHANGELOG.md]: @dcousens Fixed.

**Comment by junderw** [comment]
@dcousens Anything else?

**Comment by dcousens** [comment]
Nope,  LGTM

**Comment by junderw** [comment]
@dcousens OK

I need an approving review from a member besides myself, then I'll merge.

Could you approve it?

**Comment by junderw** [comment]
I went ahead and published to npm since there has been no objections, and all issues brought up by @dcousens were fixed.

If I can get an approval I'll merge this into github as well.

**Comment by junderw** [comment]
Also, the tag v4.0.3 has been added to github as well. (This was done automatically with the commit)

**Comment by junderw** [comment]
Could someone add an approval. I've already deployed to npm and tagged the commit.

Just need to merge it.

Thanks.
@dcousens @jprichardson @fanatid

**Comment by junderw** [comment]
Verbal approval gained, so I will go ahead and merge through a new PR.

**Pull request closed without merging**